Automattic/jetpack
# Add "Getting started with Jetpack Backup" video to Assistant and "At a glance" dashboard

- Add necessary assets and urls
- Add improvements and `compact` mode to layout
- Add video to one-click-restores recommendation
- Add video to at-a-glance backup section





#### Changes proposed in this Pull Request:

* Add the "Getting started with Jetpack Backup" video to Assistant as a part of the "One-click restores" recommendation.
* Add the "Getting started with Jetpack Backup" video to the "At a glance" dashboard in the backups section when the plugin awaits server credentials

#### Jetpack product discussion
pbNhbs-1Ah-p2

#### Does this pull request change what data or activity we track or use?
No

#### Related PR (WP-Calypso)
https://github.com/Automattic/wp-calypso/pull/64658

#### Testing instructions:






* Checkout this branch `git checkout add/how-to-backup-video`
* Run the WordPress local env with the jetpack plugin
* Run jetpack watch and select `plugins > jetpack ` in order to build the new code introduced in this PR
* Ensure you've enabled Jetpack Backups feature and you haven't provided server credentials
* Go to Recommendations page (`/wp-admin/admin.php?page=jetpack/recommendations/summary`) - you should see a small card with "Getting started ..." video below "One-click restores" card:
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/8419292/174300716-0b3e059b-1edb-4a7d-960a-6bd4a4224de4.png">

* Go to At a glance dashboard (`/wp-admin/admin.php?page=jetpack#/dashboard`) - you should see the wide card with "Getting started ..." video:
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/8419292/174575976-13283d13-4e2c-4213-a157-bc15792f3651.png">

* Clicking on both CTA and video thumbnail should bring you to the YT video: https://www.youtube.com/watch?v=pS5-i5KjKTw


* Click on dismiss ("x") icon on At a glance dashboard card
* Settings should be updated and the card should not appear anymore
